src/erlcloud_ec2.erl

```diff
@@ -2415,6 +2420,178 @@ request_spot_instances(Request, Config) ->
+decode_spot_fleet_state("modifying") -> modifying.
+
+encode_termination_policy(default) -> "Default";
+encode_termination_policy(no_termination) -> "noTermination".
+
+encode_allocation_strategy(lowest_price) -> "lowestPrice";
+encode_allocation_strategy(diversified) -> "diversified".
+
+extract_describe_spot_fleet_request(Doc) ->
+    [
+     {instance_id, get_text("instanceId", Doc)},
+     {spot_instance_request_id, get_text("spotInstanceRequestId", Doc)},
+     {instance_type, get_text("instanceType", Doc)}
+    ].
+
+-spec(describe_spot_fleet_instances/1 :: (string()) -> {ok, [proplist()]} | {error, term()}).
+describe_spot_fleet_instances(SpotFleetRequestId) ->
+    describe_spot_fleet_instances(SpotFleetRequestId, default_config()).
+
+-spec(describe_spot_fleet_instances/2 :: (string(), string()) -> {ok, [proplist()]} | {error, term()}).
+describe_spot_fleet_instances(SpotFleetRequestId, NextToken) ->
+    describe_spot_fleet_instances(SpotFleetRequestId, NextToken, undefined, default_config()).
+
+-spec(describe_spot_fleet_instances/3 :: (string(), string(), pos_integer()) -> {ok, [proplist()]} | {error, term()}).
+describe_spot_fleet_instances(SpotFleetRequestId, NextToken, MaxResults) ->
+    describe_spot_fleet_instances(SpotFleetRequestId, NextToken, MaxResults, default_config()).
+
+-spec(describe_spot_fleet_instances/4 :: (string(), string(), pos_integer(), aws_config()) -> {ok, [proplist()]} | {error, term()}).
+describe_spot_fleet_instances(SpotFleetRequestId, NextToken, MaxResults, Config) ->
+    Params = [
+        {"SpotFleetRequestId", SpotFleetRequestId},
+        {"NextToken", NextToken},
+        {"MaxResults", MaxResults}
+    ],
+    case ec2_query2(Config, "DescribeSpotFleetInstances", Params, ?NEW_API_VERSION) of
+        {ok, Doc} ->
+            {ok, [extract_describe_spot_fleet_request(Item) ||
+                Item <- xmerl_xpath:string("/DescribeSpotFleetInstancesResponse/activeInstanceSet/item", Doc)]};
+        {error, _} = E -> E
+    end.
+
+-spec(modify_spot_fleet_request/3 :: (string(), non_neg_integer(), default | no_termination) -> ok | {error, term()}).
+modify_spot_fleet_request(SpotFleetRequestId, TargetCapacity, ExcessCapacityTerminationPolicy) ->
+    modify_spot_fleet_request(SpotFleetRequestId, TargetCapacity,
+        ExcessCapacityTerminationPolicy, default_config()).
+
+-spec(modify_spot_fleet_request/4 :: (string(), non_neg_integer(), default | no_termination, aws_config()) -> ok | {error, term()}).
+modify_spot_fleet_request(SpotFleetRequestId, TargetCapacity, ExcessCapacityTerminationPolicy, Config) ->
+    Params = [
+        {"ExcessCapacityTerminationPolicy", encode_termination_policy(ExcessCapacityTerminationPolicy)},
+        {"SpotFleetRequestId", SpotFleetRequestId},
+        {"TargetCapacity", TargetCapacity}],
+
+    case ec2_query2(Config, "ModifySpotFleetRequest", Params, ?NEW_API_VERSION) of
+        {ok, _Doc} -> ok;
```

Comment:
As far as I understand AWS API "return" field can only be "true". Upon any error HTTP 400 is returned and {error, term()} tuple is returned.